src/Symfony/Component/Translation/Bridge/Phrase/README.md

```diff
@@ -0,0 +1,115 @@
+Therefor the factory should be initialised with a PSR-6 compatible cache adapter.
+
+Events
+------
+
+To enable you to perform post-processing on translation values and / or keys, two events are dispatched by this provider class.
+
+**PhraseReadEvent**
+
+_After_ reading the catalogue from Phrase, the resulting `TranslatorBag` is dispatched in a `PhraseReadEvent` prior to being returned from the read method.
+
+**PhraseWriteEvent**
+
+_Before_ writing the catalogue to Phrase, the `TranslatorBag` is dispatched in a `PhraseWriteEvent`.
+
+Fine tuning your Phrase api calls
+---------------------------------
+
+You can fine tune the read and write methods of this provider by adding query parameters to your dsn configuration.
+General usage is `read|write[option_name]=value`
+
+**example:** `phrase://PROJECT_ID:API_TOKEN@default?read[encoding]=UTF-8&write[update_descriptions]=0`
+
+See tables below for available options and, if applicable, their default values.
+
+**Read**
+
+In order to read translations from Phrase the [download locale](https://developers.phrase.com/api/#get-/projects/-project_id-/locales/-id-/download) call is made to the Phrase api. This call provides the following options.
+
+| name                              |  type  |  default value   |
+|-----------------------------------|:------:|:----------------:|
+| `branch`                          | string |                  |
+| `include_empty_translations`      |  bool  |        1         |
+| `exclude_empty_zero_forms`        |  bool  |                  |
+| `include_translated_keys`         |  bool  |                  |
+| `keep_notranslate_tags`           |  bool  |                  |
+| `format_options`                  | array  | enclose_in_cdata |
+| `encoding`                        | string |                  |
+| `skip_unverified_translations`    |  bool  |                  |
+| `include_unverified_translations` |  bool  |                  |
+| `use_last_reviewed_version`       |  bool  |                  |
+| `fallback_locale_enabled`         |  bool  |        0         |
+
+ **Write**
+
+In order to write translations to Phrase the [upload](https://developers.phrase.com/api/#post-/projects/-project_id-/uploads) call is made to the Phrase api. This call provides the following options.
+
+| name                  |  type  | default value |
+|-----------------------|:------:|:-------------:|
+| `update_translations` |  bool  |       1       |
+| `update_descriptions` |  bool  |               |
+| `skip_upload_tags`    |  bool  |               |
+| `skip_unverification` |  bool  |               |
+| `file_encoding`       | string |               |
+| `locale_mapping`      | array  |               |
+| `format_options`      | array  |               |
+| `autotranslate`       |  bool  |               |
+| `mark_reviewed`       |  bool  |               |
```

Comment:
We should remove all options from the readme and keep only the links IMHO.
This will save us from having to maintain the list.